lib/morse_translator.rb

```diff
@@ -40,4 +41,25 @@ def initialize
                     " " => " "}
   end
 
+  def translate(text)
+    characters = text.chars
+    morse_chars_array = characters.map do |character|
```

Comment:
instead of assigning this to a variable to join, you could also call end.join on line 48.